build.gradle

```diff
@@ -2,7 +2,7 @@ import static org.gradle.api.JavaVersion.*
 
 buildscript {
     ext {
-        gradle_plugins_version = '5.0.4-SNAPSHOT'
```

Comment:
Releasing a new version of the plugins is done _separately_ from adding a new feature.

Reply:
OK - will revert this and create a separate PR to bump the version number (what a waste of time).

Reply:
:+1: